_posts/2023-02-22-gsoc2023.md

```diff
@@ -0,0 +1,127 @@
+| 1       | [Generalise Groups]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-1-bead-and-ring-groups)                                         | hard     | 350 hours   | Generalise concept of groups                                                                   | Python, NetworkX, Molecular modelling                  | @lilyminium, @fiona-naughton, @richardjgowers, @IAlibay, @micaela-matta, @ojeda-e, @yuxuanzhuang, @RMeli |
+| 2       | [Extend MDAnalysis Interoperability]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-2-extend-interoperability)                  | medium     | 350 hours   | Extend converters module to other relevant packages                                            | Python, Molecular Modelling                      | @lilyminium, @IAlibay, @fiona-naughton, @hmacdope, @yuxuanzhuang, @RMeli|
+| 3       | [Molecular volume and surface analysis]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-3-molecular-volume-and-surface-analysis) | medium       | 175 hours | use an existing package for molecular surface area calculations to build a new analysis module | Python, MDAnalysis.analysis, Molecular Modelling | @orbeckst, @IAlibay, @hmacdope, @yuxuanzhuang, @RMeli |
+| 4       | [Benchmarking and performance optimization]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-4-benchmarking-and-performance-optimization)                  | medium/hard     | 175 hours   |  write benchmarks for automated performance analysis and address performance bottlenecks | Python, Molecular Modelling                      | @hmacdope, @orbeckst, @ojeda-e, @RMeli |
+| 5       | [Transport property calculations]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-5-transport-property-calculations) | medium     | 350 hours   |  write analysis code to calculate physical transport properties | Python, Physics/Mathematics      | @orionarcher, @hmacdope |
+| 6       | [Implementation of parallel analysis framework]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-6-parallel-analysis) | hard     | 350 hours   |  implement parallel framework in MDAnalysis| Python, Parallel Programming, Molecular Modelling      | @yuxuanzhuang, @orbeckst, @RMeli |
+
+
+
+## Information for prospective GSoC Contributors
+
+You must meet our [own requirements] if you want to be a GSoC
+Contributor with MDAnalysis this year (read all the docs behind these
+links!). You must also meet the [eligibility criteria]. Our [GSoC
+FAQ][] collects common questions from applicants.
+
+The MDAnalysis community values diversity and is committed to providing a
+productive, harassment-free environment to every member. Our [Code of Conduct]
+explains the values that we as a community uphold. *Every member (and every
+GSoC Contributor) agrees to follow the [Code of Conduct]*.
+
+As a start to get familiar with MDAnalysis and open source development you
+should follow these steps:
+
```

Comment:
Add YouTube trailer? It's very short but might give candidates a good overview of what's MDAnalysis is about, if they don't know already.

```suggestion

### Watch the MDAnalysis Trailer

The [MDAnalysis Trailer](https://www.youtube.com/watch?v=uMAfvwFkD3o) on YouTube
is a one minute introduction to MDAnalysis.

```